packages/integration-tests/utils/replayHelpers.ts

```diff
@@ -214,9 +214,12 @@ export function shouldSkipReplayTest(): boolean {
  * Takes a replay recording payload and returns a normalized string representation.
  * This is necessary because the DOM snapshots contain absolute paths to other HTML
  * files which break the tests on different machines.
+ * Also, we need to reset any time offsets to 0 as they can differ and cause flakes.
  */
 export function normalize(obj: unknown): string {
   const rawString = JSON.stringify(obj, null, 2);
-  const normalizedString = rawString.replace(/"file:\/\/.+(\/.*\.html)"/g, '"$1"');
+  const normalizedString = rawString
+    .replace(/"file:\/\/.+(\/.*\.html)"/, '"$1"')
+    .replace(/"timeOffset":\s*\d+/, '"timeOffset": 0');
```

Comment:
m: I would prefer it if we made the redaction explicit here.

```suggestion
    .replace(/"timeOffset":\s*\d+/, '"timeOffset": [timeOffset]');
```

Reply:
With the redaction we now need to update snapshots :D